front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -359,6 +381,42 @@ export default function LabsTranscriptsIndex({
     return updateIsActive(transcriptConfigurationId, isActive);
   };
 
+  const getDefaultConfiguration = async (provider: string) => {
+    const response = await fetch(
+      `/api/w/${owner.sId}/labs/transcripts/default?provider=${provider}`
+    );
+
+    if (response.ok) {
+      const defaultConfigurationRes = await response.json();
+      const defaultConfiguration: LabsTranscriptsConfigurationResource =
+        defaultConfigurationRes.configuration;
+
+      return defaultConfiguration;
+    }
+
+    return null;
+  };
+
+  const saveApiConnection = async (
+    apiKey: string,
+    provider: string,
+    apiKeyIsEncrypted: boolean = false
+  ) => {
+    const response = await fetch(`/api/w/${owner.sId}/labs/transcripts`, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        apiKey,
+        provider,
+        apiKeyIsEncrypted,
```

Comment:
What's `apiKeyIsEncrypted`?

Reply:
Basically when a user sets up its new configuration, we duplicate the default connection - so it's just a toggle to avoid re-encrypting the already encrypted key